app/controllers/trips_controller.rb

```diff
@@ -1,17 +1,33 @@
 class TripsController < ApplicationController
-  before_action :set_trip, only: [:dashboard, :show, :edit, :update, :destroy]
+  before_action :set_trip, only: [:dashboard, :import, :show, :edit, :update, :destroy]
+  skip_before_action :verify_authenticity_token, only: [:import]
```

Comment:
We should check the authenticity token when forms are submitted. Was there a problem you ran into?